tbx/core/context_processors.py

```diff
@@ -14,11 +15,18 @@ def peoplehr_jobs_count(request):
     """
     Add the number of open jobs to the context
     """
-    job_index = JobIndexPage.objects.first()
-    if job_index:
-        peoplehr_feed = PeopleHRFeed()
-        job_count = peoplehr_feed.get_job_count(job_index.jobs_xml_feed)
-    else:
-        job_count = 0
+    CACHE_KEY = "job_count"
+    CACHE_TIMEOUT = 21600
```

Comment:
I like writing this out as seconds * minutes * hours
```suggestion
    CACHE_TIMEOUT = 60 * 60 * 6
```